README.md

```diff
@@ -38,7 +38,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
```

Comment:
📜 Consider adding a changelog section in the README to track changes between versions. This improves maintainability and helps users understand what has changed.